database/fingertips-db/Scripts/ScriptsIgnoredOnImport.sql

```diff
@@ -0,0 +1,38 @@
+﻿
```

Comment:
What is the purpose of this script?

Reply:
I think it is created when you import a script to the sqlproj - it is what is 'left over'. I don't think it is needed and had meant to delete it it - but forgot

Reply:
will delete